iTorrent/zh-Hans.lproj/Localizable.strings

```diff
@@ -304,7 +304,7 @@
 "Peers" = "用户";
 "Seeds" = "做种";
 "Leechers" = "下载";
-"Are you shure to remove this trackers?" = "你确定要删除这个 Tracker 吗？";
```

Comment:
Do not fix localization keys without replacing them in all translation files and project itself

Reply:
I noticed that the English translation file modified this Key, and this string was indeed rendered untranslated in the program.

Reply:
![IMG_2506](https://github.com/XITRIX/iTorrent/assets/125987288/3334f5b7-7cdb-41ac-a790-88d56a84e1be)

Reply:
The modifications to the English translation are located in this commit
https://github.com/XITRIX/iTorrent/commit/e9a4d3ecd9be2f93d690f581d56fba7cc0c80d5f

Reply:
Ah, sorry for that, you are right